packages/sanity/src/structure/panes/list/ListPane.tsx

```diff
@@ -22,7 +22,7 @@ export function ListPane(props: ListPaneProps) {
   return (
     <Pane
       currentMaxWidth={350}
-      data-testid="desk-tool-list-pane"
+      data-testid="structure-tool-list-pane"
```

Comment:
It may be useful to explicitly mention this change in the release notes, in case any folks are relying on this test id for their own tests.

Reply:
Good point, will add it to the release notes